.github/workflows/scripts/cherrypick.py

```diff
@@ -1,3 +1,10 @@
+# WARNING: DO NOT EDIT!
```

Comment:
I think this file could live in the scripts directory outside of .github.

Reply:
It depends on a lot of environment variables that automatically available from the CI.  In theory you could set them yourself but it would be unwieldy.

Reply:
I have run this cherrypick PR locally many times. It's how I test it out.

Reply:
Ok, I can move it.

Reply:
Do we want to change the interface at all then?  Because it's still technically specific to the CI.

Reply:
The only change here will have to be `TRAVIS_REPO_SLUG` which we'll have to change anyway since it's not set by Github Actions. The other environment variables are the same across platforms because they're set as environment variables (or secrets). And STABLE_BRANCH is set by cherrypick.sh which is supposed to act as the interface between the CI environment and cherrypick.py when cherrypick.py isn't being run directly.

Reply:
I believe the substitute for `TRAVIS_REPO_SLUG` is `GITHUB_REPOSITORY`
https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables